internal/schemautil/templates/service_user_config_schema.json

```diff
@@ -685,7 +685,7 @@
       },
       "flink_version": {
         "enum": [
-          "1.13"
+          "1.15"
```

Comment:
Oh wait, here we should preserve the old version as well, so it should be:
```
 "enum": [
          "1.13",
          "1.15"

Reply:
We don't currently supported different versions of Flink in Aiven. Thus when Flink 1.15.1 goes live in Aiven, all new services will be 1.15.1.

Is there some reason why 1.13 should be preserved here?

Reply:
When a user upgrades the TF provider and uses 1.13, she will get an error that this value is no longer valid, this is why we only remove old values only during the major releases. 

Are you saying it is impossible that we have any users on 1.13?

Reply:
I think we can ignore this problem for now.

There might still be some users running 1.13, but they'll switch to 1.15 sooner than later. And new Flink services can only be created using 1.15.